service/controller/resource/tccp/template/template_main_security_groups.go

```diff
@@ -224,6 +224,17 @@ const TemplateMainSecurityGroups = `
       FromPort: -1
       ToPort: -1
       SourceSecurityGroupId: !Ref MasterSecurityGroup
+  MasterAllowAPIInternalELBHealthCheck:
+    Type: AWS::EC2::SecurityGroupIngress
+    DependsOn:
+      - MasterSecurityGroup
+      - APIInternalELBSecurityGroup
+    Properties:
+      GroupId: !Ref MasterSecurityGroup
+      IpProtocol: "tcp"
+      FromPort: 8089
+      ToPort: 8089
```

Comment:
I saw this port being defined in the key package so it would probably be super smart to inject it in here.

Reply:
we put all the ports there as text instead of using variables, I can add it but we will have some weird inconsistency

Reply:
Ok fine. Then let's keep it like that for now.